backend/.profile

```diff
@@ -1,11 +1,20 @@
+#!/bin/bash
+
 export REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
 
+# Grab the New Relic license key from the newrelic-creds user-provided service instance
```

Comment:
@asteel-gsa I noticed that [you don't even have to have a `newrelic.ini` file, only these two env vars set](https://docs.newrelic.com/install/python/?python-non-web=non-web-no&python-docker=docker-yes&python-framework=unsupported-asgi-framework&python-server=gunicorn#pass-in-some-basic-configurations-7). So that's all I do here now. We can introduce the `newrelic.ini` file as warranted when we start customizing other stuff.